src/main/kotlin/com/exactpro/th2/check1/entities/Checkpoint.kt

```diff
@@ -0,0 +1,12 @@
+package com.exactpro.th2.check1.entities
+
+import com.datastax.driver.core.utils.UUIDs.timeBased
```

Comment:
Just to avoid using the Cassandra dependency we can use `EventUtils.generateUUID()`. It uses the time-base generation as well